main.cpp

```diff
@@ -78,81 +77,102 @@ void RunTests(string _seedsetFile, string _resultsFile)
     getline(seedsetInput, line);
 
     std::istringstream iss(line);
-    int seeds, width, height, colourCount, totalMoves;
+    int seedCount, width, height, colourCount, maxMoves;
     getline(iss, line, ',');
-    seeds = stoi(line);
+    seedCount = stoi(line);
     getline(iss, line, ',');
     width = stoi(line);
     getline(iss, line, ',');
     height = stoi(line);
     getline(iss, line, ',');
     colourCount = stoi(line);
     getline(iss, line, ',');
-    totalMoves = stoi(line);
+    maxMoves = stoi(line);
 
     int sum = 0;
+    int errorCount = 0;
     string str = "";
-    while (getline(seedsetInput, line))
+    for (int i = 0; i < seedCount; i++)
     {
+        getline(seedsetInput, line);
```

Comment:
unnecessary

Reply:
same with many of the formatting changes - was it an automatic IDE thing (specifically Board.h / Game.h)

Reply:
If you mean the for loop then no I think it's important especiallysince you know exactly how many seeds/lines you're expecting. The complexity is the same but it allows you to avoid infinite looping and crashing or causing damage. .